### PR TITLE
feat: add DinkMigrate client command

### LINKED ISSUE #563: Add support for migrating config from the Discord Rare Drop Notifier plugin
After discussions with the current maintainer of the plugin @BossHuso we'd like to add the ability to migrate the user's plugin config into Dink.

I see this being done with a user command (e.g. `::dinkmigrate`). Name might need to be more specific, or it accepts a parameter, or it popups up a dialog with user choice or something.

Once this is implemented on our end, the Discord Rare Drop Notifier plugin can prompt users to use the command to migrate their config.

Open questions:
Should we remove the config keys from the Discord Rare Drop Notifier plugins after migrating them?

Plugin repo: https://github.com/BossHuso/discord-rare-drop-notificater

### PR DESCRIPTION
**Pull request by iProdigy**
Closes #563

https://github.com/PJGJ210/Discord-Collection-Logger/issues/5 :white_check_mark: 
https://github.com/Boredska/gim-bank-discord/issues/3
https://github.com/MidgetJake/UniversalDiscordNotifier/issues/41
https://github.com/jamesdrudolph/Discord-Death-Notifications/issues/11
https://github.com/RinZJ/better-discord-loot-logger/issues/30
https://github.com/ATremonte/Discord-Level-Notifications/issues/12 :white_check_mark: 
https://github.com/jack0lantern/raidshamer/issues/2 :white_check_mark:

### CODE REVIEW

File: src/main/java/dinkplugin/SettingsManager.java
Comment (pajlada) [inline src/main/java/dinkplugin/SettingsManager.java:209]: Should we mention that you need to re-open config panel for changes to be reflected?

**Review by pajlada — APPROVED** [review]
The readme is clear & the code looks good to me. Should be a fairly simple review - builder pattern looks good for both the small & big plugins.

Just one question regarding the message after a migration, then I'm good

I have not tested this yet, which we should do somewhat thoroughly before merging this in (+ making sure maintainers have a chance to look at the issues you've created)

File: src/main/java/dinkplugin/SettingsManager.java
Comment (iProdigy) [reply src/main/java/dinkplugin/SettingsManager.java:209]: `Please close and open the plugin settings panel for these changes to be visually reflected.` should already be in chat from the `handleImport` call, but we could reiterate it since ppl struggle to read

Comment (pajlada) [reply src/main/java/dinkplugin/SettingsManager.java:209]: ah if that's already there from the import call, then we should be all good

**Review by Felanbird — APPROVED** [review]
tested various plugins importing, looks to work as expected :+1:

**Comment by iProdigy** [comment]
No objections from the other plugin authors within a (little over a) month; merging

**Pull request merged**
